### ISSUE TITLE
Search Results and Pagination - 1.4.3 Contrast Minimum - Contrast between all text and the background is not sufficient

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9724]
As a user with vision issues meaning some colours are difficult to see, contrast between text and background should be sufficient (4.5:1).

**Linked Work Item:**
#9659 

**Expected Result:**
All contrast between text and background colours should be at or above a ratio of 4.5:1. This allows users with various forms of colourblindness and other vision issues to perceive all text.

**Actual Result:**
None of the text in the search results and pagination components meets the ratio between text and background colours.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/beae2d1f-225d-4dac-a04f-90b08b17e5bb)

**Comment by imogenroberts** [comment]
RETEST: 1.4.3 Contrast Minimum - All text and background colours in the search results now has sufficient contrast. However, two contrast errors remain on the white-on-blue search text in the search field and on the grey-on-white 'x' button used to get rid of the user's search term. They have a contrast level of 2.64:1 and 2.84:1 respectively.

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/170078e3-25b9-4d79-b0b7-70e0241e6e47)

**Comment by johnatawnclementawn** [comment]
As of 2023/09/25, I'm not seeing any further contrast issues

**Comment by imogenroberts** [comment]
See #9901 for evidence of fix.
